__tests__/unit/lib/utils/repoGitDiff.test.js

```diff
@@ -3,7 +3,7 @@ const repoGitDiff = require('../../../../lib/utils/repoGitDiff')
 const child_process = require('child_process')
 jest.mock('child_process', () => ({ spawnSync: jest.fn() }))
 jest.mock('fs-extra')
-jest.mock('fs')
+//jest.mock('fs')
```

Comment:
Is it really working without this mock ?
If yes just remove the line plz

Reply:
Yes it did! 

Ran all tests & passed - needed it to test ignore loading a file properly. Didn't seem to cause any other issues 

will remove.